src/CoordinateSubject.java

```diff
@@ -8,10 +10,14 @@
 	// update the observer
 	// read about the observer pattern
 	//
+	ArrayList<ShipElement> element = new ArrayList<ShipElement>();
```

Comment:
This is not what the tests describe. We only have one observer, but utilize the observer pattern with `this.observer` because it points to multiple elements in an array.